### DISCUSSION TITLE
Macbook keyboard cannot handle simultaneous keypresses

### DISCUSSION THREAD

**Discussion by hburn7** [ppy/osu#21894] [Q&A] [answered]
On an M1 Pro Macbook Pro 16" I am unable to play mania notes that have 3-4 simultaneous keypresses. _Sometimes_ it can handle 3, but 4 never register simultaneously. Even 2 simultaneous presses sometimes do not register correctly. This isn't osu!'s fault, I'm wondering if there's any way to fix this on my laptop. In the video here I am clicking all of the keys correctly.

https://user-images.githubusercontent.com/38370573/209852387-485ab878-c48e-4b42-93fe-79fa036240a7.mp4

**Comment by mk56-spn** [comment, marked as answer]
This is not the place for general keyboard tech support
